modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/image_selector/js/ImageSelector.js

```diff
@@ -27,16 +31,29 @@ import ProgressWrapper from './ProgressWrapper';
 const CSS_DROP_ACTIVE = 'drop-active';
 const CSS_PROGRESS_ACTIVE = 'progress-active';
 const STATUS_CODE = Liferay.STATUS_CODE;
+
+/**
+ * @deprecated As of Cavanaugh (7.4.x), replaced by imageSelectorCoverImageAtom
+ */
 const STR_IMAGE_DELETED = 'coverImageDeleted';
+
+/**
+ * @deprecated As of Cavanaugh (7.4.x), replaced by imageSelectorCoverImageAtom
+ */
 const STR_IMAGE_SELECTED = 'coverImageSelected';
+
+/**
+ * @deprecated As of Cavanaugh (7.4.x), replaced by imageSelectorCoverImageAtom
+ */
```

Comment:
Before I forward this to Lima to take a look, opinions on whether these events need to go through a full deprecate-then-remove cycle? I am not sure whether customers might be listening to them, because I don't really know how or why they would be overriding the image selector. If you think there is no risk of them being used (even though they are globally-visible events so, in theory, anything is possible), it would be cool to just remove them.

@ambrinchaudhary @jbalsas

Reply:
> Before I forward this to Lima to take a look, opinions on whether these events need to go through a full deprecate-then-remove cycle?

![killitwithfire](https://user-images.githubusercontent.com/905006/114875020-ca818900-9df4-11eb-8db9-b0c0db8a8c57.gif)

Reply:
For future reference and those that can't read gifs, I agree that it's highly unlikely that anything outside Liferay is using and wiring the component the way we are so removing it feels like a good move.

To be on the safe side, we can add an entry in `BREAKING_CHANGES.md` notifying of this and outlining the upgrade process (similar to what we're doing here) in the off-chance someone out there is using the tag and listening to those events in a custom development.

Reply:
Sounds good to me. I was hoping that deletion would be the answer... 😈